site/src/pages/RoadmapPage/sidebar/ProgramRequirementsList.tsx

```diff
@@ -26,19 +26,33 @@ import { ProgramRequirement } from '@peterportal/types';
 import { setGroupExpanded, setMarkerComplete } from '../../../store/slices/courseRequirementsSlice';
 import { getMissingPrerequisites } from '../../../helpers/planner';
 import { useClearedCourses } from '../../../hooks/planner';
-import { useTransferredCredits } from '../../../hooks/transferCredits';
+import { useTransferredCredits, TransferredCourseWithType } from '../../../hooks/transferCredits';
 import { useIsLoggedIn } from '../../../hooks/isLoggedIn';
-
 import ArrowRightIcon from '@mui/icons-material/ArrowRight';
 import ArrowDropDownIcon from '@mui/icons-material/ArrowDropDown';
+import SwapHorizOutlinedIcon from '@mui/icons-material/SwapHorizOutlined';
+
+interface SourceOverlayProps {
+  completedBy: TransferredCourseWithType['transferType'] | 'roadmap' | null;
+}
+const SourceOverlay: FC<SourceOverlayProps> = ({ completedBy }) => {
+  if (!completedBy || completedBy === 'roadmap') return null;
+  const title = `Cleared by ${completedBy === 'AP' ? 'an AP Exam' : 'a transferred course'}`;
+  const icon = completedBy === 'AP' ? 'AP' : <SwapHorizOutlinedIcon style={{ fontSize: 16 }} />;
```

Comment:
because I'm a hater of inline styles, I believe you should instead write `.MuiSvg-root { font-size: 16px; }` inside of the `.source-overlay` rule.